tilings/strategies/experimental_verification.py

```diff
@@ -0,0 +1,122 @@
+"""
+This file contains "experimental" verification strategies, which are those that verify
+tilings for which we are not certain we can independently calculate their counting
+sequence or generating function.
+"""
+
+from typing import Iterable, Iterator, Optional
+
+from comb_spec_searcher import StrategyFactory, VerificationStrategy
+from permuta import Basis, Perm
+from tilings import GriddedPerm, Tiling
+from tilings.algorithms import SubclassVerificationAlgorithm
+
+__all__ = [
+    "SubclassVerificationStrategy",
+]
+
+TileScopeVerificationStrategy = VerificationStrategy[Tiling, GriddedPerm]
+
+
+class SubclassVerificationStrategy(TileScopeVerificationStrategy):
+    """
+    A strategy object for holding the results of SubclassVerification
+    """
+
+    def __init__(
+        self, subclass_basis=Iterable[Perm], ignore_parent: bool = True,
+    ):
+        self.subclass_basis = tuple(sorted(subclass_basis))
+        super().__init__(ignore_parent=ignore_parent)
+
+    @staticmethod
+    def verified(tiling: Tiling) -> bool:
+        return True
```

Comment:
I'd take @christianbean opinion but this feels weird to me. Should we check that the strategy applies?